src/restapi.rs

```diff
@@ -1,12 +1,89 @@
 use crate::utils::DisplayVec;
-use goose::goose::{GooseUser, TransactionResult};
+use goose::goose::{GooseMethod, GooseRequest, GooseUser, TransactionError, TransactionResult};
+use reqwest::{Client, RequestBuilder};
+
+use crate::utils::GooseUserData;
+use rand::Rng;
 use serde_json::json;
 use std::sync::{
     Arc,
     atomic::{AtomicUsize, Ordering},
 };
 use urlencoding::encode;
 
+/// Get advisory total count
+pub async fn get_advisory_total(user: &mut GooseUser) -> TransactionResult {
```

Comment:
Please don't do this is a transaction, but once on the startup of the application (if required) and pass in the value as `u64` to functions.